pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1131 @@
+	}
+}
+
+func (Binding) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "Binding ties one object to another - for example, a pod is bound to a node by a scheduler.",
+		"metadata": "Standard object's metadata. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"target":   "Target is the object to bind to.",
+	}
+}
+
+func (Capabilities) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":     "Capabilities represent POSIX capabilities that can be added or removed to a running container.",
+		"add":  "Added capabilities",
+		"drop": "Removed capabilities",
+	}
+}
+
+func (ComponentCondition) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":        "ComponentCondition contains information for the condition of a component.",
+		"type":    "Type of component condition, currently only Healthy.",
+		"status":  "Current status of this component condition, one of True, False, Unknown.",
+		"message": "Health check message received from the component.",
+		"error":   "Error code from health check attempt (if any)",
+	}
+}
+
+func (ComponentStatus) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":           "ComponentStatus (and ComponentStatusList) holds the cluster validation info.",
+		"metadata":   "Standard object's metadata. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata",
+		"conditions": "List of component conditions observed",
+	}
+}
+
+func (ComponentStatusList) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "ComponentStatusList is a list of ComponentStatus.",
+		"metadata": "Standard list metadata. See http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds",
+		"items":    "List of component status objects.",
+	}
+}
+
+func (Container) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":                       "Container represents a single container that is expected to be run on the host.",
+		"name":                   "Name of the container. This must be a DNS_LABEL. Each container in a pod must have a unique name. Cannot be updated. Required.",
+		"image":                  "Docker image name. Optional. See http://releases.k8s.io/HEAD/docs/user-guide/images.md",
+		"command":                "The docker image's entrypoint is used if this is not provided. Entrypoint array. Not executed within a shell. Cannot be updated. Variable references $(VAR_NAME) are expanded using the container's environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Optional. See http://releases.k8s.io/HEAD/docs/user-guide/containers.md#containers-and-commands",
```

Comment:
Change the order of the first two sentences. Currently the first sentence gives supplementary information and the second sentence states what "command" is.